src/app/services/admin-patient.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import { ServerApiService } from './server-api.service';
+import { Observable } from 'rxjs';
+
+export class GenericAdminPatientResponse {
+  public message: string;
+}
+
+export class ResetPasswordResponse {
+  public password: string;
+}
+
+export class UserListItem {
+  public name: string;
+  public type: string;
+}
+
+@Injectable()
+export class AdminPatientService {
+  constructor(private serverApiService: ServerApiService) { }
+
+  public createUser(name: string, password: string, type: string) {
+    return this.serverApiService.post<GenericAdminPatientResponse>('/user/create', {'name': name, 'password': password, 'type': type});
```

Comment:
Refer to Griffin's merged branch: https://github.com/movshov/VA-Audiology-App/pull/42
Service should hit the endpoint `/accounts/create`